cartaya/message-page/resources/css/styles.css

```diff
@@ -0,0 +1,75 @@
+@font-face {
+    font-family: 'Lato-Regular';
+    src: url('./../font/Lato-Regular.eot?#iefix') format('embedded-opentype'),
+         url('./../font/Lato-Regular.woff') format('woff'),
+         url('./../font/Lato-Regular.ttf') format('truetype'),
+         url('./../font/Lato-Regular.svg#FontName') format('svg');
+}
+
+.view-port {
+    align-items: center;
+    display: flex;
+    height: 100vh;
+    justify-content: center;
+}
+
+.lato-font {
+    font-family: 'Camphor-Thin', 'Arial', sans-serif;
```

Comment:
font-face is defining lato-regular. Here you are declaring camphor-thin...which one it is?